app/src/main/res/values/strings.xml

```diff
@@ -183,8 +175,7 @@
         Santa Monica, CA - Wants long-term / engineer / no kids / yogi, adventurer / must like
         dogs\n
         \n
-        Remember, only an about Twitter sizes message will show up during profile comparison. Of
-        course, you can add more text for actual matches.
+       Of course, you can add more text for actual matches.
```

Comment:
This was edit it so much that the actual message was lost.

The paragraph had two points:
1. The profile (shown to potential matches) is short. The user has to diligently choose what the user writes in it.
2. The profile is also shown to actual matches. (The actual matches see a full screen profile view instead of the half screen comparison version.) In that full screen version, the full length of the profile is shown.

In effect, the user can choose to write the brief 1.5 line profile. That will show up on the profile comparison screen to potential matches. Additionally, the user could add more profile text. That won't show up during comparison, but will be visible later. It's a bit of a way to have a pseudo private section of the profile that's only visible to actual matches.

In other words, the main contents is:
1. Choose your words carefully only a short Twitter-size message is visible during profile comparison.
2. Feel free to add additional text. Your actual matches will be able to see that.